Description: Fix FTBFS on hurd
 sys/sysctl.h is not implemented on hurd, so don't include it if we're
 building Irrlicht on hurd.
Author: Vincent Cheng <vcheng@debian.org>
Last-Update: 05-29-2012

--- a/source/Irrlicht/COSOperator.cpp
+++ b/source/Irrlicht/COSOperator.cpp
@@ -13,9 +13,11 @@
 #include <unistd.h>
 #ifndef _IRR_SOLARIS_PLATFORM_
 #include <sys/types.h>
+#if !defined(__gnu_hurd__)
 #include <sys/sysctl.h>
 #endif
 #endif
+#endif
 
 #if defined(_IRR_COMPILE_WITH_X11_DEVICE_)
 #include "CIrrDeviceLinux.h"
